spec/features/upload_descriptive_metadata_spec.rb

```diff
@@ -17,7 +17,7 @@
     "title1.value,purl\nmy title,https://purl.stanford.edu/#{Druid.new(item.externalIdentifier).without_namespace}\n"
   end
   let(:file) do
-    Tempfile.new('upload.csv').tap do |file|
+    Tempfile.new(%w[upload .csv]).tap do |file|
```

Comment:
Note: using an array in this call honors the file extension, so now the tempfile will look like `upload2022-05-02.csv` instead of `upload.csv2022-05-02` which causes an unsupported filetype error on upload.